src/NuGet/Microsoft.CodeAnalysis.CSharp.Workspaces.nuspec

```diff
@@ -27,6 +27,8 @@ Supported Platforms:
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\portable-net45+win8" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\net45" />
     <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.xml" target="lib\net45" />
+    <file src="Microsoft.CodeAnalysis.CSharp.Workspaces.dll" target="lib\netstandard1.3" />
```

Comment:
When you add support for netstandard you also need to list out the dependencies.